package.json

```diff
@@ -17,6 +17,7 @@
     "url": "git://github.com/angular/material.git"
   },
   "private": true,
+  "main": "./dist/angular-material.js",
```

Comment:
I recently had to install angular-material from a specific commit for a patch (https://github.com/angular/material/commit/bd1cce42ba16b733af0163011f8bb2429782b5da) that is not included in the latest npm package.  My project uses webpack to bundle all dependencies together and, without the main field, it was unable to resolve the module.

I was unaware of bower-material.  It looks like switching to that module will also solve the webpack issue.